src/plugins/mf_priority.cpp

```diff
@@ -768,6 +768,16 @@ static void reprior_cb (flux_t *h,
         goto error;
     if (flux_respond (h, msg, NULL) < 0)
         flux_log_error (h, "flux_respond");
+
+    // iterate through map that stores associations and held job IDs; check to
+    // see if any previously-held jobs can now be released with the update
+    for (auto &entry: users) {
+        auto &banks = entry.second;
+
+        for (auto &bank_entry : banks) {
+            check_and_release_held_jobs (p, &bank_entry.second);
```

Comment:
Should error code be checked here? If purposefully ignoring error code, I suggest casting the function to `void` to indicate this is intentional:

```c
    (void) check_and_release_held_jobs (p, &bank_entry.second);
```